openedx/core/djangoapps/content/course_overviews/migrations/0023_courseoverview_banner_image_url.py

```diff
@@ -13,11 +13,11 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='courseoverview',
             name='banner_image_url',
-            field=models.TextField(null=True),
+            field=models.TextField(),
```

Comment:
why wasn't this handled with a newer migration?

Reply:
The reason that this wasn't handled with a newer migration is because the upstream pull request was merged. So we didn't want to result in more code drift.